Sources/Orbit/Components/CountryFlag.swift

```diff
@@ -28,6 +28,10 @@ public struct CountryFlag: View {
         }
     }
 
+    var imageName: String {
+        UIImage(named: countryCode.lowercased(), in: .current, with: nil) != nil ? countryCode.lowercased() : "unknown"
```

Comment:
Even though I'm sure this is cached, it's still unfortunate that we need to try to create an image just to see if it is bundled... #63 might help with this 🤔